Apps/Playground/UWP/App.cpp

```diff
@@ -129,7 +129,7 @@ void App::Uninitialize()
     }
 
     m_chromeDevTools.reset();
-    m_inputBuffer.reset();
+    nativeInput = nullptr;
```

Comment:
nit: Inconsistent way to reset pointer. Earlier it was this:
```suggestion
    nativeInput = {};
```

Reply:
Changed